src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Sort.txt

```diff
@@ -38,4 +38,40 @@
 [true,true,false,false,Blank()]
 
 >> First(LastN(Sort([-2, -1, 0, 1, 2], 1 / Value), 3)).Value
-#Error
\ No newline at end of file
+#Error
+
+>> SortByColumns([-2, -1, 5, 1, 2], "Value")
```

Comment:
>SortByColumns [](http://example.com/codeflow?start=3&length=13)

nit - can you place in its own SortByColumns.txt file?

Reply:
Moved!